index.html

```diff
@@ -222,6 +259,35 @@ <h1>🗺 Not on the map yet?</h1>
         }
       }
 
+      function infoWindow(dataIndex) {
+        const { name, position, slack } = state.people[dataIndex];
+        const content = document.createElement("div");
+
+        content.className = "map-info-window";
+        content.innerHTML =
+          '<h1></h1><dl class="info"><dt>Slack</dt><dd class="slack"></dd><dt>Distance</dt><dd class="distance"></dd></dl><p class="directions"><a>Get directions</a></p>';
+        content.querySelector("h1").textContent = name;
+        content.querySelector("dd.slack").textContent = slack;
+
+        if (state.location) {
+          // overestimate the distance to compensate for obstacles. This is super
+          // scientific, obviously
+          const dist = (1.22 * distance(state.location, position)).toFixed(1);
+          const text = `about ${dist} km`;
+
+          content.querySelector("dd.distance").textContent = text;
+          content
+            .querySelector("p.directions a")
+            .setAttribute(
+              "href",
+              `https://maps.google.com?daddr=${position.lat},${position.lng}`
+            );
+        } else {
+        }
```

Comment:
Do we want to hide `dd.distance` in the `else` branch? I.e. 
```js
content.querySelector("dt.distance").remove(); // need to give the <dt> element the class
content.querySelector("dd.distance").remove();
```

Reply:
Yea, good shout